src/verifier/az_snp_vtpm/mod.rs

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use super::{Attestation, TeeEvidenceParsedClaim, Verifier};
+use anyhow::{anyhow, Context, Result};
+use async_trait::async_trait;
+use az_snp_vtpm::certs::{get_chain_from_amd, get_vcek_from_amd};
+use az_snp_vtpm::hcl::{buf_to_hcl_data, HclReportWithRuntimeData};
+use az_snp_vtpm::report::Validateable;
+use az_snp_vtpm::vtpm::{Quote, VerifyVTpmQuote};
+use openssl::sha::sha256;
```

Comment:
instead of bringing a new openssl crate dependency, can we use the `sha2` crate (which we already use for Sha384) to handle Sha256?

Reply:
oh right 👍